modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+* <<http/http-connector.adoc#input-sources, HTTP Connector>>
+* xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)]
+
+To determine whether the connector you are working with has its own input sources, see the reference guide for that connector.
+
+=== Authorization credentials
+
+Connectors that connect to external APIs use the Basic (Username and Password), OAuth, or OAuth2 authentication scheme.
+
+To configure authorization credentials for a connector, it is recommended that you create a <<global-element,global element>>, which enables you to reuse the configuration for other connector instances. For some connectors, configuring a global element is mandatory.
+
+=== Connector Operations
+
+Connector operations are calls to external APIs or other connectivity-related actions that a connector can perform. You can use connector operations anywhere in a flow.
+
+Each connector has different supported operations. To view a list of supported operations for a specific connector, see the reference guide for that connector.
+
+[[best-practices]]
+== Best Practices for Configuring Connectors
+
+The following best practices apply to configuring connectors:
+
+* Use a global element to define configuration details that can be shared with other instances of the connector in the flow.
+* Use property placeholders for property values if you are configuring a connector in Studio or configuring it using XML outside the Studio editor.
+* Test the connectivity of a connector before you deploy it.
+* Configure a reconnection strategy to set the number of reconnection attempts to try after an initial failure.
+
+[[global-element]]
+=== Use a Global Element to Define Configuration Details
+
+Global elements act as reusable objects. When you configure a global element for a connector, other instances of that connector in the flow can share that global element.
+
+To use a global element, create one global element to configure your connection, configuration, or transport details for the connector. Reference the global element from other instances of the connector in the flow to ensure consistency across flow elements.
+
+In addition to the benefits of reusability and consistency, global elements also make connector maintenance easier. If you must change a connector's configuration, you can change the configuration once in the global element and it will apply to all connectors that reference the global element.
+
+Most connectors support the use of a global element for authentication and other functions. For information about configuring a global element for a specific connector, see the user guide for that connector.
+
+[[property-placeholders]]
+=== Use Property Placeholders for Property Values
+
+Instead of using static values for a connector configuration in Studio, you can use http://ant.apache.org[Ant-style] property placeholders, as shown in this Studio example:
```

Comment:
```suggestion
Instead of using static values for a connector configuration in Anypoint Studio, you can use http://ant.apache.org[Ant-style] property placeholders, as shown in this Studio example:
```